.code-samples.meilisearch.yaml

```diff
@@ -357,7 +351,7 @@ getting_started_add_documents_md: |-
     var movies []map[string]interface{}
     json.Unmarshal(byteValue, &movies)
 
-    update, err := client.Index("movies").AddDocuments(movies)
+    _, err := client.Index("movies").AddDocuments(movies)
```

Comment:
In some samples you leave the variable even if it is not used in the same asmple 

For example: 

```
faceted_search_update_settings_1: |-
  response, err := client.Index("movies").UpdateAttributesForFaceting(&[]string{
    "director",
    "genres",
  })
```

How do you decide when you `_` the variable ?

Reply:
You're right I think it better to leave the name everywhere.
Also, I realize that sometimes it's `result` and sometimes `response`.

Reply:
Ah but this one it's the getting started. So I change it to be ready to use if you copy-paste the code

Reply:
I think you're right!